project2/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Project 2: C O M P S </title>
+    <link href="https://fonts.googleapis.com/css?family=Be+Vietnam:300,600,800&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+    <section>
+        <img src="images/featured-film-example.jpg" class="articlepicture"
+            alt="Everything is Illuminated movie screenshot, guy staring at a wall with hanging objects">
+        <div>
+            <h2 class="articlename"><a href="#"> This is the title of my article i can make it linger let see
+                    whthappenhereif i write more </a></h2>
+            <ul class="clearfix">
+                <li class="authorinfo"> <a href="#"> By Juanita</a></li>
+                <li class="articledate"> <a href="#">02.02.012</li> </a>
+            </ul>
+            <p class="abstract"> Lorem, ipsum dolor sit amet consectetur adipisicing elit. Corrupti ad harum facere nemo
+                consequatur sunt
+                beatae eos magni recusandae exercitationem iusto earum repellat quibusdam, rerum illum dolore
+                consequuntur
+                blanditiis quos! <span class="readmore"><a href="readmore.com">Read more</a></span> </p>
+            <p class="tags"> <a href="lin1k.com"> #tags </a> <a href="link2.com"> #tags2 </a> <a href="link3.com">
```

Comment:
1. Consider using another `ul` for the tags? Is it not a list of tags?
2. Avoid using made up links like "lin1k.com" because, if you haven't tested them out, you might accidentally link to something in your comp that is spam or worse. Better to use either `"#"`, as you did above or known URLs like `"https://google.com"`.